controllers/replication-controller/persistentvolume_controller_test.go

```diff
@@ -473,9 +473,19 @@ func TestPVReconcileSuite(t *testing.T) {
 	suite.Run(t, testSuite)
 }
 
+func (suite *PVReconcileSuite) TestPVNotFound() {
+	req := suite.getTypicalRequest("non-existent-pv")
+	_, err := suite.reconciler.Reconcile(context.Background(), req)
+	suite.NoError(err, "No on PV reconcile")
```

Comment:
If no PV is found, then there should be an error returned. In that case, we should be checking for suite.Error().